maa-cli/Cargo.toml

```diff
@@ -42,10 +42,10 @@ futures-util = { version = "0.3.28" }
 sha2 = { version = "0.10.7" }
 digest = { version = "0.10.7" }
 semver = { version = "1.0.19", features = ["serde"] }
-
 # Dependencies used to extract files
 flate2 = { version = "1", optional = true }
 tar = { version = "0.4.40", optional = true }
+prettytable = "0.10.0"
```

Comment:
The `prettytable` crate depends on `csv` by default, providing functions for importing and exporting CSV files, which we don't use. I suggest we use the non-default feature. Additionally, we can consider moving this dependency higher in the list, as this group includes some optional dependencies.